src/web_peer_connection.cpp

```diff
@@ -699,7 +699,7 @@ void web_peer_connection::handle_redirect(int const bytes_left)
 		// the new web seed we're adding only has this file for now
 		// we may add more files later
 		web->redirects[file_index] = redirect_path;
-		if (web->have_files.get_bit(file_index) == false)
+		if (!web->have_files.get_bit(file_index))
```

Comment:
I'm not convinced about everything this checker suggests. `a == false` can be a lot clearer and easier to read than `!a`, especially when `a` is a long expression

Reply:
On the other hand, using `a == false` can lead to accidentally writing `a = false` sometimes, leading  to possibly very hard to track bugs, Unless one commits to using the `false == a` style.